configs/setups/fesom-recom/fesom-recom.yaml

```diff
@@ -204,14 +119,19 @@ recom:
         model_dir: "${general.model_dir}/recom/"
         setup_dir: "${general.model_dir}"
 
-        yearly_outputs_start: 1             # Year in which the yearly output starts to be saved
         time_step: "${fesom.time_step}"
 
-
         choose_lresume:
                 true:
                         choose_general.run_number:
                                 1:      # Restart from runscript
                                         ini_parent_exp_id: "${general.ini_parent_exp_id}"
                                         ini_restart_dir: "${general.ini_restart_dir}/recom/"
                                         ini_parent_date: "${prev_date}"
+
+        # setup-specific namelist.recom 
+        config_sources:
+                recom: "${namelist_dir}/namelist.recom"
+        add_config_in_work:
+                recom: "namelist.recom"
```

Comment:
Delete this, this is the `recom` namelist and should come from the `recom.yaml` or the `recom` section in this file